site/docs/v1/tech/installation-guide/kickstart.adoc

```diff
@@ -275,6 +275,28 @@ In this example I have a license Id of `eb7244dc-5d8e-40cd-a005-70b116fbda31`.
 
 If you are using the community edition, you can omit this field.
 
+=== Timeouts
+
+[NOTE.since]
+====
+Available since 1.20.1
+====
+
+Kickstart works by making API calls. By default each API request has a read and a connect timeout, and the API call will fail if it does not succeed within the timeout period. 
+
+You can configure these timeouts with a different value. For example, if you have high latency between the FusionAuth instance and its database, you may need to increase these values.
+
+You can do so by creating a toplevel `settings` object, and setting either or both of the `connectTimeout` and `readTimeout` attributes. The value is specified in milliseconds.
+
+```json
+"settings": {
+  "connectTimeout": 7000,
+  "readTimeout": 7000
+},
+```
+
+The default value for each of these, if unset, is 5 seconds.
+
```

Comment:
Should we structure this like the API JSON request body doc? 

Then we'd still have an example JSON of the settings object - just like you have here, but we'd use a table fomat to document each parameter.

Reply:
Sounds good.